common/src/main/java/com/netflix/conductor/common/metadata/workflow/WorkflowDef.java

```diff
@@ -273,4 +273,19 @@ public int hashCode() {
 				getSchemaVersion()
 		);
 	}
+
+	@Override
+	public String toString() {
+		return "WorkflowDef{" +
+				"name='" + name + '\'' +
```

Comment:
**Nit:** I am guessing this is auto-generated code?  It would be nice to not do so much string creation via `+`.  But maybe the JIT can optimize it away?

Reply:
Yeah, autogenerated code. I think JIT will take care of that, however I don't see why not fix it at code level.
Will use `MoreObjects.toStringHelper` from guava, i think it is the way to go on this one.